tmt/steps/prepare/feature/__init__.py

```diff
@@ -171,7 +171,21 @@ class PrepareFeature(tmt.steps.prepare.PreparePlugin[PrepareFeatureData]):
          before using ``feature`` plugin from an alternative repository
          or local build.
 
-    Control enablement of various features on the guest:
+    The ``feature`` plugin provides a comfortable way to
+    enable and disable some commonly used functionality on the guest.
+    As for now enabling and disabling the ``epel`` repository is
+    supported, ``crb`` and ``fips`` are coming in the near
+    future.
+
+    Example config:
+
+    .. code-block:: yaml
+
+        prepare:
+            how: feature
+            epel: enabled
+
+    Or
```

Comment:
Actually, these paragraphs are no longer true, and we have an improved documentation for `prepare/feature` and its plugins, see https://tmt.readthedocs.io/en/latest/plugins/prepare.html#feature. So, I'd say the content moved here no longer adds anything new - see the example below, it's a richer version of what you just transferred from specs; my two cents, we can leave `prepare/feature` as it is, and removing its bits from specs should be enough.